src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -326,4 +329,8 @@ private void addScmDetails() {
             return Collections.emptyList();
         }
     }
+
+    private String getDisplayName() {
+        return run.hasCustomDisplayName() ? run.getDisplayName() : job.getFullDisplayName();
```

Comment:
Perhaps this would be better.

```java
private String getDisplayName() {
    return run.hasCustomDisplayName() ? job.getFullDisplayName() + " » " + run.getDisplayName() : job.getFullDisplayName();
```